src/fastpm.c

```diff
@@ -384,31 +383,10 @@ prepare_cosmology(FastPMCosmology * c, RunData * prr) {
 
     int i;
     double m_ncdm_i;
-    double m_ncdm_sum = 0;
     for(i = 0; i < CONF(prr->lua, n_m_ncdm); i ++) {
         m_ncdm_i = CONF(prr->lua, m_ncdm)[i];
         c->m_ncdm[i] = m_ncdm_i;
```

Comment:
Could make these into 1 line now and remove the double m_ncdm_i; if you like

Reply:
Yes. Done.